### DISCUSSION TITLE
Unable to write to Folder Settings because no resource is provided.

### DISCUSSION THREAD

**Discussion by blaylockbk** [coder/code-server#3789] [Q&A] [answered]
I just updated to 3.11.0 and get this error every time I open a window.

```
Unable to write to Folder Settings because no resource is provided.
```

Not sure how to debug this notice or what is causing it. Nothing in the banner indicates it's a specific extension, but maybe it is. 
![image](https://user-images.githubusercontent.com/6249613/125971757-9b8cbcaf-3ad0-45c4-a8f9-109850d77192.png)

Any tips?

**Comment by jsjoeio** [comment, marked as answer]
That's very very strange 🤔

## Debugging

Here are [some tips](https://coder.com/docs/code-server/v3.11.0/FAQ#how-do-i-debug-issues-with-code-server) but is there anything in the browser console or the server logs (i.e. where code-server is running).

> Nothing in the banner indicates it's a specific extension, but maybe it is

Can you try disabling all extensions and seeing if the issue still appears?

**Reply by blaylockbk** [reply]
Well, I reinstalled code-server and all the extensions, I'm not getting the banner anymore.

**Reply by blaylockbk** [reply]
I'm mostly sure that it was related to this issue with Jupyter: https://github.com/cdr/code-server/issues/3782

**Reply by jsjoeio** [reply]
Well that's good!

Yeah, we need to look more into that. Thanks for reporting back!

**Reply by blaylockbk** [reply]
This might be the problem...
```[2021-07-26 17:12:07.113] [remoteagent] [error] /p/home/blaylock/anaconda3/envs/vscode/share/code-server/extensions/ms-toolsai.jupyter-2021.6.0 Extension is not compatible with Code 1.57.1. Extension requires: 1.56.0.```

code-server's jupyter extension is v2021.6.832593372 but the latest in the real vscode store is v2021.8.1054968649

**Reply by jsjoeio** [reply]
Hmm... we might need to manually upgrade then. Let's check with @oxy who has solved this in the past.
